Formula/l/llvm.rb

```diff
@@ -60,6 +60,9 @@ def install
               /^(\s*library_path\s*=\s*)None$/,
               "\\1'#{lib}'"
 
+    # fix https://github.com/llvm/llvm-project/issues/106361 until 19.1.0 final is released with the fix
+    inreplace "llvm/lib/Support/Z3Solver.cpp", "#include <set>", "#include <set>\n#include <unordered_map>"
```

Comment:
We should use the merged patch here. You can find examples of doing this by searching for `patch do` in this repository.

Reply:
Sure, but I don't want to reset the queue position for the Linux runner. Final 19.1.0 is expected within 24 hrs anyway.

Reply:
Final got delayed to September 17, but rc4 has the fix and macOS is green.